tests/testthat/test-params.R

```diff
@@ -0,0 +1,61 @@
+# params -------------------------------------------------------------------
+
+test_that("NULL values are stripped", {
+  expect_equal(params(), set_names(list()))
+})
+
+test_that("checks its inputs", {
+  expect_snapshot(error = TRUE, {
```

Comment:
Have never thought to intentionally trigger many errors inside of one `expect_snapshot()` call😮